_maps_2024/11-jack.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 11
+creator: Ji Jin (Jack) Chen
+title: Arctic
+thumbnail: /assets/thumbnails/2024/11-Jack-Arctic.png
+source: https://github.com/jijinjc/30DayMappingChallenge2024/blob/main/Arctic-11.Rmd
+---
+
+For the arctic map, I wanted to revisit an old friend of mine: Normalized Difference Snow Index (NDSI). I wanted to find a way to get NDSI data and map the NDSI for an Arctic region, and truly see how much snow there is. Luckily, it would seem that there is still a lot of snow in Nunavut, and even the water was frozen when the tif image was created. This data was actually collected through Google Earth Engine
```

Comment:
```suggestion
For the arctic map, I wanted to revisit an old friend of mine: Normalized Difference Snow Index (NDSI). I wanted to find a way to get NDSI data and map the NDSI for an Arctic region, and truly see how much snow there is. Luckily, it would seem that there is still a lot of snow in Nunavut, and even the water was frozen when the tif image was created. This data was actually collected through Google Earth Engine

![Arctic map](assets/thumbnails/2024/11-Jack-Arctic.jpg)
```